src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3115,6 +3115,16 @@ void ApplicationManagerImpl::Handle(const impl::MessageFromHmi message) {
     return;
   }
 
+  Json::Value value;
+  Json::Reader reader;
+  reader.parse(message->json_message(), value);
+  if (value.isMember(("result"))) {
```

Comment:
@LitvinenkoIra May be it is worth to add the same check for erroneous response from hmi? error also must be an object according to specification

Reply:
@AByzhynar Done d3d9c53035ce23e08e856bc71878bce9595404a5